### PR TITLE
#trivial - `f-checkout` Simplify storybook knobs

### PR DESCRIPTION
**Pull request by FayeCarter**
### Changed
- Storybook knobs to be easier to use.

~ Before | After |
| --- | --- |
| image | image |

### CODE REVIEW

**Review by LTurns — DISMISSED** [review]

File: packages/components/organisms/f-checkout/stories/Checkout.stories.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue';
 import Vuex from 'vuex';
-import { select, text } from '@storybook/addon-knobs';
+import { select, text, boolean } from '@storybook/addon-knobs';
```
Comment (dandel10n) [inline packages/components/organisms/f-checkout/stories/Checkout.stories.js]: text one is not in use anymore so can be removed

**Review by siggerzz — DISMISSED** [review]
Looks great! Thank you for this 🎉

**Comment by xander-marjoram** [comment]
Nice stuff, happy to approve once Maryia's suggestion has been fixed

**Review by DamianMullins — COMMENTED** [review]
It looks like `getCheckoutCollectionUrl`, `getBasketDeliveryUrl`, and `getBasketCollectionUrl` variables no longer exist but are still referenced when calling `setupCheckoutMethod`?

**Review by DamianMullins — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

**Pull request merged**